server/vulnerabilities/msrc/analyzer.go

```diff
@@ -139,14 +140,13 @@ func patched(
 			}
 
 			isGreater, err := winBuildVersionGreaterOrEqual(build, os.KernelVersion)
```

Comment:
If `isGreater` is `true`, should it stop the inner for loop and continue with the next `KBID`?

Reply:
there _should_ only be a single OS match within the `v.RemediatedBy` array, so the assumption here is that if a match is found we can return from the function. It may be worth adding tests to `generate` to ensure that's always the case.